### ISSUE TITLE
Fix npm audit errors (e.g. braces)

### ISSUE THREAD

**Issue by dagadbm** [anastasiia-zolochevska/react-appinsights#44]
There is an npm audit error regarding braces.
Since react-appinsights depends on braces this should be checked to see the effort to update braces to >= 2.3.1.

Thank you

**Comment by pviotti** [comment]
Thanks for reporting this. I've just released v2.0.2, which updates the dependencies. However, unfortunately, some audit errors (including perhaps the one you reported) are still there as they are due to some dependencies that haven't been maintained in a long time, such as `appinsights-usage` and `rollup-plugin-node-builtins`. Below are the details. In the upcoming [v3.x](https://www.npmjs.com/package/react-appinsights/v/3.0.0-beta.3) we are going to address that by dropping not actively maintained dependencies. 

```
Moderate        Regular Expression Denial of Service
Package         semver
Patched in      >=4.3.2
Dependency of   rollup-plugin-node-builtins [dev]
Path            rollup-plugin-node-builtins > browserify-fs > levelup >
                  semver
More info       https://npmjs.com/advisories/31

Moderate        Memory Exposure
Package         bl
Patched in      >=0.9.5 <1.0.0 || >=1.0.1
Dependency of   rollup-plugin-node-builtins [dev]
Path            rollup-plugin-node-builtins > browserify-fs > levelup > bl
More info       https://npmjs.com/advisories/596

Low             Regular Expression Denial of Service
Package         braces
Patched in      >=2.3.1
Dependency of   appinsights-usage
Path            appinsights-usage > babel-cli > chokidar > anymatch >
                 micromatch > braces
More info       https://npmjs.com/advisories/786
```

**Comment by pviotti** [comment]
This has been fixed in the master branch and it's released as v3.0.0-rc.6 in the beta channel.
